src/shared/constants/status.js

```diff
@@ -12,7 +12,7 @@ export const MENTOR_ANSWERS = [
 export const IDENTIFIERS = [
   {
     value: false,
-    label: 'Active Duty Veteran / Spouse'
+    label: 'Servicemember, Veteran, Military Spouse / Dependent'
```

Comment:
This label is too long, and won't display entirely on mobile.

Reply:
Hmm maybe 'Military / Veteran / Spouse'?

Reply:
Sounds good to me.